Description: Force LC_* environment variables when testing (non) l10n'isation.
 Permits building in non-"C" environments.
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Last-Update: 2012-09-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cups/testppd.c
+++ b/cups/testppd.c
@@ -494,6 +494,14 @@
     * Test localization...
     */
 
+   /*
+    * Enforce void localization
+    */
+    putenv("LANG=C");
+    putenv("LC_ALL=C");
+    putenv("LC_CTYPE=C");
+    putenv("LC_MESSAGES=C");
+
     fputs("ppdLocalizeIPPReason(text): ", stdout);
     if (ppdLocalizeIPPReason(ppd, "foo", NULL, buffer, sizeof(buffer)) &&
         !strcmp(buffer, "Foo Reason"))
